asmcli/commands/help.sh

```diff
@@ -175,6 +175,8 @@ FLAGS:
 
      --managed                        Provision a remote, managed control plane
                                       instead of installing one in-cluster.
+     --legacy                         Provision a remote, managed control plane
+                                      with the legacy asmcli installer.
```

Comment:
Can you add "that runs client-side" somewhere in the description?

Reply:
Done.